service/history/configs/quotas.go

```diff
@@ -37,65 +37,65 @@ const (
-		"GetShard":                               1,
-		"DeleteWorkflowExecution":                1,
-		"DescribeHistoryHost":                    1,
-		"DescribeMutableState":                   1,
-		"DescribeWorkflowExecution":              1,
-		"GetDLQMessages":                         1,
-		"GetDLQReplicationMessages":              1,
-		"GetMutableState":                        1,
-		"GetReplicationMessages":                 1,
-		"ImportWorkflowExecution":                1,
-		"IsActivityTaskValid":                    1,
-		"IsWorkflowTaskValid":                    1,
-		"MergeDLQMessages":                       1,
-		"PollMutableState":                       1,
-		"PurgeDLQMessages":                       1,
-		"QueryWorkflow":                          1,
-		"ReapplyEvents":                          1,
-		"RebuildMutableState":                    1,
-		"RecordActivityTaskHeartbeat":            1,
-		"RecordActivityTaskStarted":              1,
-		"RecordChildExecutionCompleted":          1,
-		"VerifyChildExecutionCompletionRecorded": 1,
-		"RecordWorkflowTaskStarted":              1,
-		"RefreshWorkflowTasks":                   1,
-		"RemoveSignalMutableState":               1,
-		"RemoveTask":                             1,
-		"ReplicateEventsV2":                      1,
-		"ReplicateWorkflowState":                 1,
-		"RequestCancelWorkflowExecution":         1,
-		"ResetStickyTaskQueue":                   1,
-		"ResetWorkflowExecution":                 1,
-		"RespondActivityTaskCanceled":            1,
-		"RespondActivityTaskCompleted":           1,
-		"RespondActivityTaskFailed":              1,
-		"RespondWorkflowTaskCompleted":           1,
-		"RespondWorkflowTaskFailed":              1,
-		"ScheduleWorkflowTask":                   1,
-		"VerifyFirstWorkflowTaskScheduled":       1,
-		"SignalWithStartWorkflowExecution":       1,
-		"SignalWorkflowExecution":                1,
-		"StartWorkflowExecution":                 1,
-		"SyncActivity":                           1,
-		"SyncShardStatus":                        1,
-		"TerminateWorkflowExecution":             1,
-		"GenerateLastHistoryReplicationTasks":    1,
-		"GetReplicationStatus":                   1,
-		"DeleteWorkflowVisibilityRecord":         1,
-		"UpdateWorkflowExecution":                1,
-		"PollWorkflowExecutionUpdate":            1,
-		"StreamWorkflowReplicationMessages":      1,
-		"GetWorkflowExecutionHistory":            1,
-		"GetWorkflowExecutionHistoryReverse":     1,
-		"GetWorkflowExecutionRawHistoryV2":       1,
-		"ForceDeleteWorkflowExecution":           1,
-		"GetDLQTasks":                            1,
-		"DeleteDLQTasks":                         1,
-		"AddTasks":                               1,
-		"ListQueues":                             1,
+		"/temporal.server.api.historyservice.v1.HistoryService/CloseShard":                             1,
+		"/temporal.server.api.historyservice.v1.HistoryService/GetShard":                               1,
```

Comment:
Did you check for other places we are using method names? I saw some quotas tests that use method name that we might want to update just for consistency.

Reply:
I updated the quotas tests.